src/com/owncloud/android/ui/errorhandling/ErrorMessageAdapter.java

```diff
@@ -55,267 +53,118 @@
-                // Error --> No permissions
-                message = String.format(res.getString(R.string.forbidden_permissions),
-                        res.getString(R.string.update_link_forbidden_permissions));
-
-            }
-
-        } else if (operation instanceof MoveFileOperation) {
-
-            if (result.getCode() == ResultCode.FILE_NOT_FOUND) {
-                message = res.getString(R.string.move_file_not_found);
-            } else if (result.getCode() == ResultCode.INVALID_MOVE_INTO_DESCENDANT) {
-                message = res.getString(R.string.move_file_invalid_into_descendent);
-
-            } else if (result.getCode() == ResultCode.INVALID_OVERWRITE) {
-                message = res.getString(R.string.move_file_invalid_overwrite);
-
-            } else if (result.getCode() == ResultCode.FORBIDDEN) {
-                message = String.format(res.getString(R.string.forbidden_permissions),
-                        res.getString(R.string.forbidden_permissions_move));
-
-            } else if (result.getCode() == ResultCode.INVALID_CHARACTER_DETECT_IN_SERVER) {
-                message = res.getString(R.string.filename_forbidden_charaters_from_server);
-
-            } else if (result.getCode() == ResultCode.CONFLICT) {
-                message = res.getString(R.string.move_file_error);
-            }
-
-        } else if (operation instanceof SynchronizeFolderOperation) {
-
-            if (!result.isSuccess()) {
-                String folderPathName = new File(
-                        ((SynchronizeFolderOperation) operation).getFolderPath()).getName();
-                if (result.getCode() == ResultCode.FILE_NOT_FOUND) {
-                    message = String.format(
-                            res.getString(R.string.sync_current_folder_was_removed),
-                            folderPathName
-                    );
-                }
-            }
-
-        } else if (operation instanceof CopyFileOperation) {
-            if (result.getCode() == ResultCode.FILE_NOT_FOUND) {
-                message = res.getString(R.string.copy_file_not_found);
-
-            } else if (result.getCode() == ResultCode.INVALID_COPY_INTO_DESCENDANT) {
-                message = res.getString(R.string.copy_file_invalid_into_descendent);
-
-            } else if (result.getCode() == ResultCode.INVALID_OVERWRITE) {
-                message = res.getString(R.string.copy_file_invalid_overwrite);
+    public static String getErrorCauseMessage(final RemoteOperationResult result,
+                                           final RemoteOperation operation,
+                                           final Resources resources) {
+        Formatter f = new Formatter(resources);
+        if(result.isSuccess()) {
+            if(operation instanceof UploadFileOperation) return f.format(R.string.uploader_upload_succeeded_content_single, ((UploadFileOperation) operation).getFileName());
```

Comment:
There are some lines that exceed 120 columns, which is the limit we use. 

You can check or even update that limit in Android Studio by going to File -> Settings -> Other settings -> Editor -> Code Style and setting the right margin (columns) to 120. 

When your right margin is properly configured please split the lines that exceed that limit into two different ones.